tests/conftest.py

```diff
@@ -86,6 +86,7 @@
 
 def pytest_addoption(parser):
     parser.addoption("--testbed", action="store", default=None, help="testbed name")
+    parser.addoption("--testbed_idx", action="store", default=None, help="testbed name index")
```

Comment:
Don't think testbed_idx is a good name of parameter here,
since this name makes no sense to sonic-mgmt repo.
Do we have some parameters like log_file or so to be used here to pass with testbed_idx?

Reply:
Good point, I'll try to find another approach